pycaption/dfxp/base.py

```diff
@@ -58,10 +60,92 @@ def detect(self, content):
         else:
             return False
 
+    def format_timestamp(self, value):
+        datetime_value = timedelta(milliseconds=(int(value / 1000)))
+        str_value = text_type(datetime_value)[:11]
+        if not datetime_value.microseconds:
+            str_value += '.000'
+        return '0' + str_value
```

Comment:
Is this for adding a leading 0 in the hour? Shouldn't it check for the length first?

Reply:
I repurposed it from https://github.com/hearsttv/pycaption/blob/master/pycaption/base.py#L232.